dojson/contrib/marc21/fields/bd84188x.py

```diff
@@ -29,13 +37,13 @@ def holding_institution(self, key, value):
     }
 
 
-@marc21.over('location', '^852[_103254768][10_2]')
+@marc21.over('location', '^852[_12345678][_012]')
 @utils.for_each_value
 @utils.filter_values
 def location(self, key, value):
     """Location."""
     indicator_map1 = {
-        "#": "No information provided",
```

Comment:
not a major thing but might be worth it to replace " with ' to make the code follow the conventions used elsewhere